application.py

```diff
@@ -102,7 +110,37 @@ def post(self):
         return 'OK', 201
 
 
+class AnnotationList(Resource):
+    def post(self):
+        query = annotation_query_parser.parse_args()
+
+        db_query = db.session.query(Event)
+
+        start_date = datetime.strptime(query['range']['from'], ISO_DATE_STRING)
+        end_date = datetime.strptime(query['range']['to'], ISO_DATE_STRING)
+        category = query['annotation']['query'].split(',')
+
+        db_query = db_query.filter(Event.category.in_(category))
+
+        db_query = db_query.filter(Event.unix_timestamp >= calendar.timegm(start_date.timetuple()))
+        db_query = db_query.filter(Event.unix_timestamp <= calendar.timegm(end_date.timetuple()))
+
+        result = db_query.order_by(Event.unix_timestamp.desc()).all()
+
+        events = [
+            {
+                "annotation": query['annotation'],
+                "time": r.unix_timestamp * 1000,
+                "title": "Criticality {criticality}".format(criticality=r.criticality),
+                "tags": r.category,
```

Comment:
Shouldn't `tags` be an array? In which case, we could include the criticality here; maybe something like `criticality:$CRIT` and `category:$CAT`, to avoid (unlikely) namespace collisions.